src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+
+  for (let i = 0; i < keys.length; i++) {
+    if (values.slice(i + 1).includes(values[i])) {
+      return null;
+    }
+  }
+
+  const inverse = {};
+
+  for (let i = 0; i < values.length; i++) {
+    inverse[values[i]] = keys[i];
+  }
```

Comment:
Do you really need two loops here? Also `slice` and `includes` used loops too
You can just use one `for ... in` loop and read from object by key to check if this property is already in object